src/Queries/QueryBuilder/QueryBuilder.ts

```diff
@@ -773,19 +780,28 @@ export class QueryBuilder {
   };
 
   /**
-   * returns an object with replacing its values with a bind param value
-   * example return value: ( a.p1 = $v1, b.p2 = $v2 )
+   * returns an object with replacing its values with a bind param value.
+   * if value is a literal, returns literal name as value
+   * example return value: { a.p1 = $v1, b.p2 = $v2, c.p3 = literalP3 }
    */
   public static getPropertiesWithParams = (
     /** data to set */
-    data: AnyObject,
+    data: Neo4jSupportedProperties,
     /** bind param to use and mutate */
     bindParam: BindParam,
   ): string => {
     const parts: string[] = [];
 
     for (const key of Object.keys(data)) {
-      parts.push(`${key}: $${bindParam.getUniqueNameAndAdd(key, data[key])}`);
+      if (!data.hasOwnProperty(key)) {
+        continue;
+      }
```

Comment:
Is this needed? :)

Reply:
you're right! removed it in [6a50eb0](https://github.com/themetalfleece/neogma/pull/99/commits/6a50eb0e14931ea448d0c73d700d7ef83523b56d)

Reply:
you're right! removed it in [c74dda](https://github.com/themetalfleece/neogma/pull/99/commits/7c74dda829c7088a0eae4b210a9b642cb0c32de5)